segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/BookKeeperLog.java

```diff
@@ -634,6 +635,7 @@ private void tryTruncate(LedgerAddress upToAddress) {
      * @throws DataLogInitializationException If an Exception (other than NoNodeException) occurred.
      */
     @VisibleForTesting
+    @VisibleForDebugging
```

Comment:
I think this should replace visableForTesting here.

Reply:
The idea was that the method is publicly visible for both of those reasons. I wanted to keep that in there.